core/setup.py

```diff
@@ -23,7 +23,8 @@
     'cryptography==3.3.2',
     'pyyaml>=5.4.1, <6.0',
     'requests>=2.23.0, <3.0',
-    'Deprecated>=1.2.13, <1.3'
+    'Deprecated>=1.2.13, <1.3',
+    'luddite==1.0.2'
```

Comment:
It is better to use version ranges, maybe pin it <1.1 - so we can get latest patch versions.